octobot_backtesting/time/channel/time_updater.py

```diff
@@ -35,21 +35,27 @@ async def start(self):
         self.channels_manager = channels_manager.ChannelsManager(exchange_ids=self.backtesting.exchange_ids,
                                                                  matrix_id=self.backtesting.matrix_id)
         await self.channels_manager.initialize()
+        latest_progress = 0
+        last_print = time.time()
         while not self.should_stop:
             try:
                 current_timestamp = self.time_manager.current_timestamp
                 await self.push(self.time_manager.current_timestamp)
-
-                self.logger.info(f"Progress : {round(min(self.backtesting.get_progress(), 1) * 100, 2)}% "
-                                 f"[{current_timestamp}]")
+                if self.backtesting.get_progress() - latest_progress > 0.05:
+                    latest_progress = self.backtesting.get_progress()
+                    print(f"Progress : {round(min(self.backtesting.get_progress(), 1) * 100, 2)}% "
```

Comment:
I'm not fond of using `print`. Could we use something else? Or use a logger that is not disabled?

Reply:
this should'nt have been commited, I'll remove it